packages/phrases/src/locales/en/translation/admin-console/subscription/quota-item.ts

```diff
@@ -142,13 +142,14 @@ const quota_item = {
     name: 'MFA',
     limited: 'MFA',
     unlimited: 'MFA',
-    not_eligible: 'Disable your MFA',
+    /** UNTRANSLATED */
+    not_eligible: 'Remove your MFA',
```

Comment:
update the key name?

Reply:
No, this is used in the not eligible to downgrade table.